ha/core/system_health/bootstrap/json_kv_convertor.py

```diff
@@ -43,85 +52,158 @@ def __init__(self):
 
 
     # collct kvs and if all 3 values for a component are avilable, generate healthEvent
-    def _update_health(self, key, val):
-
+    def _update_health(self, key: str, val: str) -> None:
+        """
+           Get the required uid, last_updated abd helath status value.
+           Cleans up the key. Ex: node.compute.hw.cpu.health.status will
+           be node.compute.hw.cpu. If health status value is appropriate,
+           calls the API to create health event with above and additional
+           conf index and conf store object parameters
+        """
         if re.search(self._filter_list[0].replace(':',''), key):
             self._uid = val
         elif re.search(self._filter_list[1].replace(':',''), key):
             self._last_modified = val
         elif re.search(self._filter_list[2].replace(':',''), key):
             self._status = val
+            if self._status.upper() in ["NA", "UNKNOWN", "N/A"]:
```

Comment:
We should create event even if status is received as NA / UNKNOWN etc

Reply:
Sure. Done